site/documentation/content/admin-guide/hono-client-configuration.md

```diff
@@ -25,7 +25,8 @@ The variable names contain `${PREFIX}` as a placeholder for the particular *comm
 | `${PREFIX}_HOST`<br>`--${prefix}.host` | no | `localhost` | The IP address or name of the host to connect to. NB: This needs to be set to an address that can be resolved within the network the adapter runs on. When running as a Docker container, use Docker's `--network` command line option to attach the local container to the Docker network that the service is running on. |
 | `${PREFIX}_HOSTNAME_VERIFICATION_REQUIRED`<br>`--${prefix}.hostnameVerificationRequired` | no | `true` | A flag indicating whether the value of the `${PREFIX}_HOST` variable must match the *distinguished name* or any of the *alternative names* asserted by the server's certificate when connecting using TLS. |
 | `${PREFIX}_IDLE_TIMEOUT`<br>`--${prefix}.idleTimeout` | no | `16000` | Sets the amount of time in milliseconds after which a connection will be closed when no frames have been received from the remote peer. This property is also used to configure a heartbeat mechanism, checking that the connection is still alive. The corresponding heartbeat interval will be set to *idleTimeout/2* ms. |
-| `${PREFIX}_INACTIVE_LINK_TIMEOUT`<br>`--${prefix}.inactiveLinkTimeout` | no | `0` | Sets the amount of time in milliseconds after which a AMQP sender link will be closed when no message has been sent on it. Links will not automatically be closed if the value is `0`. |
+| `${PREFIX}_INACTIVE_LINK_TIMEOUT`<br>`--${prefix}.inactiveLinkTimeout` | no | `-1` | Sets the amount of time after which an AMQP sender link will be closed when no message has been sent on it. Links will not be closed automatically if the value is `-1`. |
+| `${PREFIX}_INACTIVE_LINK_TIMEOUT_UNIT`<br>`--${prefix}.inactiveLinkTimeoutUnit` | no | minutes | Sets the time unit for the inactive link timeout. Valid values are the enum constants of [Java's TimeUnit](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/TimeUnit.html), e.g. "HOURS". |
```

Comment:
we should put single back quotes around the default value as we do for the other values as well.